assets/javascripts/anser-import.js

```diff
@@ -0,0 +1,5 @@
+const module = {};
```

Comment:
Is this a workaround for something?

Reply:
Yes, the library is apparently supposed to be used within Node.js and therefore populates `module.exports` to integrate with its module system. We currently don't use a bundler like Browserify so I thought it was the easiest to simply create this variable before the inclusion. The rest of the JS code itself just works within the browser so we can get away with it. (Like I said in the chat, in the long term we likely need to use some kind of bundler because more JS libraries are now distributed this way. However, for the sake of including this small library it seemed unnecessary.)